ujt/callbacks.py

```diff
@@ -82,17 +97,20 @@ def update_graph_elements(
         virtual_node_update_signal: String used as a signal to indicate that the virtual node addition/deletion was valid. 
         collapse_n_clicks_timestamp: Timestamp of when the collapse button was clicked. Value unused, input only provided to register callback.
         expand_n_clicks_timestamp: Timestamp of when the expand button was clicked. Value unused, input only provided to register callback.
+        override_dropdown_value: Status enum value of the status to override for the node.
+        
         state_elements: The list of current cytoscape graph elements. This is unused and can be removed in a later change. 
         selected_node_data: The list of data dictionaries for selected nodes. Used to create virtual nodes.
         virtual_node_input_value: The value of the virtual node input box. Used to perform all virtual node operations. 
+        tap_node: The cytoscape element of the latest tapped node. Used check which node to override the status of. 
     Returns:
         A dictionary of cytoscape elements describing the nodes and edges of the graph.
     """
 
     ctx = dash.callback_context
     triggered_id, triggered_prop, triggered_value = utils.ctx_triggered_info(ctx)
 
-    print(triggered_id, triggered_prop, triggered_value)
+    # print(triggered_id, triggered_prop, triggered_value)  # for debugging...
```

Comment:
probably as part of later refactoring PRs, getting rid of these types of comments. for now its fine.

Reply:
Yeah, definitely will do towards the end! I left this one in for now since it's used so often.

There are definitely a few places I'll have to change to productionize / produce a version ready for "release".

Reply:
You might want to tag everything you're going to get rid of with an easily searchable string (I like "DEBUG_REMOVE") so you don't miss anything when you go to do the cleanup.